gax/src/main/java/com/google/api/gax/rpc/ServerStreamingAttemptCallable.java

```diff
@@ -335,14 +335,26 @@ private void onRequest(int count) {
 
   /** Called when the inner callable has responses to deliver. */
   private void onAttemptResponse(ResponseT message) {
+    message = resumptionStrategy.processResponse(message);
+
+    // ResumptionStrategy suppressed the message
+    if (message == null) {
+      // Since the current request is being suppressed, a new one needs to be requested from the
+      // inner callable to replace.
+      if (!autoFlowControl) {
+        innerController.request(1);
+      }
+      return;
+    }
```

Comment:
This feels odd to me - you are using a null response to signal that you don't like the response you got, so make a new request?